workshops/management/commands/report_all_instructor_activity.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import yaml
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+from workshops.models import Award, Badge, Person, Qualification, Role, Task
+
+
+class Command(BaseCommand):
+    args = ''
+    help = 'Report all instructor activity.'
```

Comment:
I'd like more explaining help message, ie. what exactly does this command report.